iframe.html

```diff
@@ -27,6 +27,8 @@
     var docUrl = undefined;
     if (!!queryParams.match(/docUrl=([^&]*)/)) {
       playerParam["docUrl"] = decodeURIComponent(queryParams.match(/docUrl=([^&]*)/)[1]);
+    } else {
+      playerParam["docUrl"] = document.referrer;
```

Comment:
I've seen some customer complain about referrer being null. Can you put in an extra check? If it is output something to the console so we can see it.